src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -436,6 +445,11 @@ internal Accessibility LocalAccessibility
         /// </summary>
         internal bool LocalDeclaredReadOnly => (DeclarationModifiers & DeclarationModifiers.ReadOnly) != 0;
 
+        /// <summary>
+        /// Indicates whether this accessor has a 'field' keyword.
```

Comment:
>has [](http://example.com/codeflow?start=44&length=3)

To be precise, this property indicates that we might have a ```field``` keyword. We just have an IdentifierExpressionSyntax, but we don't know yet how it resolves.

Reply:
Updated the doc comment.